edb/server/compiler_pool/pool.py

```diff
@@ -452,6 +452,19 @@ def sync_worker_state_cb(
 
         return preargs, callback
 
+    async def _acquire_worker(self):
+        while (
+            worker := await self._workers_queue.acquire()
```

Comment:
Can this loop complete with no connected worker found?

Reply:
I don't think so, it'll block until a connected one is found I think.